conjure-java-jersey-server/src/main/metrics/conjure-server-metrics.yml

```diff
@@ -0,0 +1,12 @@
+options:
+  javaPackage: com.palantir.conjure.java.server.jersey
+  javaVisibility: packagePrivate
+namespaces:
+  conjure.server:
+    shortName: ConjureServer
+    docs: Conjure server metrics
+    metrics:
+      exception:
```

Comment:
So I think it would be good to align this will all the other existing server-side metrics we have in WC, e.g. `server.response.500`, `server.response.error`, `server.response.all`.

- I'd like to include the `.all` suffix to emphasise that this isn't tagged by the endpoint, it's everything.
- I'd like to include a `server.response` prefix to emphasise that these are errors encountered while trying to produce a response (as distinct from background tasks or config deserialization etc).

thoughts on calling this something like `server.response.internalerror.all`?

Reply:
Good call, I think that name makes sense.